community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/resume.py

```diff
@@ -533,18 +536,34 @@ def _allocate_nodes_to_placements(nodes: List[str], excl_job_id:Optional[int], l
 
     return placements
 
+def calculate_chunk_size(max_distance, node):
+    # Calculates the chunk size based on max distance value received
+    machine_type = node.machine_type
+    if max_distance == 1:
+        return 22
+    elif max_distance == 2:
+        if machine_type.startswith("a3"):
```

Comment:
After https://github.com/GoogleCloudPlatform/cluster-toolkit/pull/3407 lands, you can get family from `template.machine_type.family`

Reply:
Updated